pkg/network/setup/netstat.go

```diff
@@ -66,23 +66,19 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		return nil
 	}
 
-	if len(vmi.Status.Interfaces) == 0 {
```

Comment:
I seems that [this]( https://github.com/kubevirt/kubevirt/blob/ff19275b8883d14356b350b8729ee965a312b7d7/pkg/network/setup/netstat.go#L142) code can be removed now.

Reply:
I think you are right, and I was doing so but...
The problem is that the current unit tests are not well covering all these scenarios, and I found myself deleting a lot of code without being sure it is safe or not.

My strategy is now to introduce unit test which we can trust (#6951) in order to remove and refactor most of the dead code that existed. I think it is safer this way.